sqlalchemy_continuum/utils.py

```diff
@@ -90,6 +90,30 @@ def parent_class(version_cls):
     return get_versioning_manager(version_cls).parent_class_map[version_cls]
 
 
+def parent_table(table):
+    versioning_manager = get_versioning_manager(table)
+    if table in versioning_manager.association_version_tables:
+        parent_table = next(
+            iter(
+                t
+                for t in versioning_manager.association_tables
```

Comment:
Can we change `association_tables` -> `association_tables_map` ?
That way we can do a simple lookup here instead of `versioning_manager.options['table_name'] % t.name == table.name`

Reply:
Hi @AbdealiJK ,

added a lookup dict to capture association version_tables